data/sql_updates/create_totals_house_senate_view.sql

```diff
@@ -7,7 +7,7 @@ with last as (
     )
         cmte_sk,
         two_yr_period_sk,
-        coh_bop as cash_on_hand_end_period,
+        coh_cop as cash_on_hand_end_period,
```

Comment:
We check 2 fields for that because filers don't report that consistently 

Example:
openFEC/data/sql_updates/create_reports_house_senate_view.sql
```
coalesce(coh_cop_i, coh_cop_ii) as cash_on_hand_end_period,
```